.github/workflows/copilot-chat-tests.yml

```diff
@@ -0,0 +1,86 @@
+name: Copilot Chat Tests
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main", "feature*" ]
+    paths: 
+      - 'samples/apps/copilot-chat-app/**'
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    defaults:
+      run:
+        working-directory: samples/apps/copilot-chat-app/webapp
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+        cache-dependency-path: samples/apps/copilot-chat-app/webapp/yarn.lock
+        cache: 'yarn'
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 6.0.x
+
+    - name: Install dependencies
+      run: yarn install
+
+    - name: Install Playwright Browsers
+      run: yarn playwright install --with-deps
+
+    - name: Update AIService configuration
+      working-directory: samples/apps/copilot-chat-app/webapi
+      env:
+        AzureOpenAI__ApiKey: ${{ secrets.AZUREOPENAI__APIKEY }}
+        AzureOpenAI__Endpoint: ${{ secrets.AZUREOPENAI__ENDPOINT }}
+      run: |
+        dotnet dev-certs https
+        dotnet user-secrets set "AIService:Key" "$AzureOpenAI__ApiKey"
+        dotnet user-secrets set "AIService:Endpoint" "$AzureOpenAI__Endpoint"
+
+    - name: Start service in background
+      working-directory: samples/apps/copilot-chat-app/webapi
+      run: |
+        dotnet run > service-log.txt 2>&1 &
+        for attempt in {0..20}; do
+          jobs
+          echo 'Waiting for service to start...';
+          if curl -k https://localhost:40443/healthz; then
+            echo;
+            echo 'Service started';
+            break;
+          fi;
+
+          sleep 5;
+        done
+
+    - name: Run Playwright tests
+      env:
+        REACT_APP_BACKEND_URI: https://localhost:40443/
+        REACT_APP_AAD_CLIENT_ID: ${{ secrets.COPILOT_CHAT_REACT_APP_AAD_CLIENT_ID }}
```

Comment:
What app registration did you end up using? Let's communicate it with the team so no one makes changes to it

Reply:
Same with test user